src/qoiread.c

```diff
@@ -0,0 +1,59 @@
+#include "proto.h"
+
+#define pixelcolor(px) (((uint32_t) (px).a << 24) | ((uint32_t) (px).b << 16) | ((uint32_t) (px).g << 8) | (uint32_t) (px).r)
```

Comment:
I know this is basically `PLUM_COLOR_VALUE_32`, but (a) then I'd have to do `PLUM_COLOR_VALUE_32(px.r, px.g, px.b, px.a)`, and (b) none of your src files actually use any of the convenience macros in header/color.h anyway.